components/composition/Grid/Grid.ts

```diff
@@ -0,0 +1,98 @@
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+const directions = ['column', 'row-reverse', 'column-reverse'];
+
+const justify = [
+  'flex-end',
+  'center',
+  'space-between',
+  'space-around',
+  'space-evenly',
+];
+
+const align = ['flex-end', 'center'];
+
+const gap = ['xs', 'sm', 'lg', 'xl'];
+
+@customElement('diamond-grid')
+export class Grid extends LitElement {
+  @property({ type: Boolean }) readonly wrap?: boolean;
```

Comment:
Could wrap type either be string to match [the CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap): `'nowrap' | 'wrap' | 'wrap-reverse'`

or just accept wrap-reverse: `boolean | 'wrap-reverse'`